CHANGELOG.md

```diff
@@ -1,13 +1,25 @@
 # App Center SDK for .NET Change Log
 
-## Version 3.2.2 (Under development)
+## Version 3.3.0
 
 ### App Center
 
 #### Android
 
 * **[Fix]** Fix reporting Xamarin.Android version.
 
+#### iOS
+
+* **[Feature]** Add support for XCFrameworks.
+* **[Improvement]** Use namespaced NSUserDefaults keys with the MSAppCenter prefix for all the keys set by the SDK. Fixed a few keys missing namespace.
+* **[Feature]** Swift Package Manager support for Analytics and Crashes modules.
```

Comment:
Both features aren't related to .NET

Reply:
@MatkovIvan I made the changes. please help to review. thanks.

Reply:
So this release should be 3.2.2?